src/Pillar-Chrysal/ConfigurationForPillar.class.st

```diff
@@ -175,7 +175,7 @@ ConfigurationForPillar >> attribution: aValue [
 { #category : #accessing }
 ConfigurationForPillar >> baseDirectory [
 	"generated code"
-	^ self propertyAt: #baseDirectory ifAbsent: [(FileSystem workingDirectory / '.')]
+	^ (FileSystem workingDirectory)
```

Comment:
1. the parenthesis are not necessary here ;).

2. Be careful, this was generated code (See the comment).
This means that if we regenerate the code, your change will be lost.
If you want that the baseDirectory does not depend anymore in the configuration (which seems a good thing), then we should just remove this method, and fix the configuration description so this method is not generated anymore.

@Ducasse could give more details about chrysal and configuration descriptions.